src/api/public/apidocs-new/paths/status_message.yaml

```diff
@@ -0,0 +1,34 @@
+get:
+  summary: Get list of status messages
+  description: |
+    Get a list of all status messages
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/limit.yaml'
```

Comment:
The `limit` parameter should be added to `https://github.com/openSUSE/open-build-service/blob/master/src/api/public/apidocs-new/paths/status_messages.yaml` as well. What can be done in another PR, of course.